architecture-building-systems/CityEnergyAnalyst
# 2192 single plot view

This PR implements #2192 - users can click on a plot title in the new (react) dashboard and a new tab opens up with an uncluttered view of the plot.

To test:
- open the dashboard for a project/scenario
- click on the title of a plot
- you should see a new tab with the plot
- try different dashboard styles (row / grid layout)
